File: py/bubble_sort.py
def bubblesort():
	for i in range(n):
		swap = 1;
		if (swap == 1):
			swap= 0;
			for j in range(n-i-1):
				if a[j]>a[j+1]:
					swap=1;
					temp=a[j];
					a[j]=a[j+1];
					a[j+1]=temp;
	return a;

n= input('Enter the limit of array');
a=[];
for i in range(n):
	b= input();
	a.append(b);
print('This is The array you have created: ');
print a;
bubblesort();
print('After sorting the list is as follows: ');
print a;

File: algo/c/bubble_sort.c
#include<stdio.h>

void bubblesort(int p[],int n)
{
	int temp,i,j;
	int swap=1;
	for(i=0;i<n&&swap==1;i++)
	{
		swap=0;
		for(j=0;j<n-i;j++)
		{
			if(p[j]>p[j+1])
			{
				swap=1;
				temp=p[j];
				p[j]=p[j+1];
				p[j+1]=temp;
			}
		}
	}
}

int main()
{
	int a[20];
	printf("\nEnter the limit of array");
	int lim;
	scanf("\n %d",&lim);
	//int a[lim];
	int i;
	for(i=0;i<lim;i++)
	{
		printf("\nEnter Value");
		scanf("\n %d",&a[i]);
	}
	printf("\nThis is The array you have created\n");
	for(i=0;i<lim;i++)
	{
		printf("%d\t",a[i]);
	}
	printf("\n Starting to sort your array via bubble sorting ");
	bubblesort(a,lim);
	printf("\nAfter sorting the list is as follows:");
	for(i=0;i<=lim;i++)
	{
		printf("\n%d",a[i]);
	}
return(0);
}
